model_compression_toolkit/quantizers_infrastructure/inferable_infrastructure/pytorch/activation_quantization_holder.py

```diff
@@ -0,0 +1,57 @@
+# Copyright 2023 Sony Semiconductor Israel, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# ==============================================================================
+
+from model_compression_toolkit.constants import FOUND_TORCH
+from model_compression_toolkit.logger import Logger
+from model_compression_toolkit.quantizers_infrastructure import BaseInferableQuantizer
+
+if FOUND_TORCH:
+    import torch
+
+    class PytorchActivationQuantizationHolder(torch.nn.Module):
```

Comment:
Inconsistency with the name in Keras which is called **ActivationQuantizationHolder** without **Keras**.

Reply:
@reuvenperetz will fix this in Keras